builder/builder-next/worker/label/label.go

```diff
@@ -0,0 +1,9 @@
+package label
+
+// Pre-defined label keys similar to BuildKit ones
+// https://github.com/moby/buildkit/blob/master/worker/label/label.go
+const (
+	prefix = "org.mobyproject.buildkit.worker."
+
+	HostGatewayIP = prefix + "hostgatewayip"
```

Comment:
I think the "magic" value is `host-gateway`; should we use hyphens here as well, or are those not "allowed"? (e.g. `host-gateway-ip`)

If this label is specific to the `moby` worker, should it have a prefix for that? Looks like BuildKit uses `containerd.` as prefix for containerd-worker-specific options; https://github.com/moby/buildkit/blob/2951a28cd7085eb18979b1f710678623d94ed578/worker/label/label.go#LL14C34-L15C59

```go
ContainerdUUID      = prefix + "containerd.uuid"      // containerd worker: containerd UUID
ContainerdNamespace = prefix + "containerd.namespace" // containerd worker: containerd namespace
```

Reply:
> e.g. `host-gateway-ip`)

SGTM

> If this label is specific to the `moby` worker, should it have a prefix for that?

:+1: